lib/validateSignature.ts

```diff
@@ -45,13 +45,13 @@ const hasValidSignature = (xml, cert, certThumbprint) => {
   signed.keyInfoProvider = {
```

Comment:
Is this still supported? The documentation seems to have changed a bit relating to this.https://github.com/node-saml/xml-crypto#verifying-xml-documents

Reply:
It seems to work, atleast from the tests we have but worth checking to see if we can replace it with the new way.

Reply:
That's in a future version